src/parser/xasl_generation.c

```diff
@@ -9789,6 +9790,38 @@ pt_to_single_key (PARSER_CONTEXT * parser, PT_NODE ** term_exprs, int nterms, bo
 	    }
 	}
 
+      if (multi_col_pos[i] != -1)
+	{
+	  /* case of multi column term. case1 : value type, case2 : function type */
+	  if (pt_is_set_type (rhs) && PT_IS_VALUE_NODE (rhs))
+	    {
+	      rhs = rhs->info.value.data_value.set;
+	    }
+	  else if (pt_is_set_type (rhs) && PT_IS_FUNCTION (rhs) && rhs->info.function.function_type == F_SEQUENCE)
+	    {
+	      rhs = rhs->info.function.arg_list;
+	    }
+	  else
+	    {
+	      /* rhs must be set type and (value or function type) */
+	      goto error;
+	    }
+	  for (pos = 0; pos < multi_col_pos[i]; pos++)
+	    {
+	      if (!rhs && pt_is_set_type (rhs))
```

Comment:
check if it is proper condition. "!rhs" and pt_is_set_type would be always false or access violation for !rhs.

Reply:
yes it was wrong. I fix it